### PR TITLE
[improve][doc] Include Elasticsearch sink compatibility 

### LINKED ISSUE #15514: [Doc] Specify which ElasticSearch and Opensearch versions are compatible with Pulsar IO ElasticSearch Sink
### What issue do you find in Pulsar docs?

In the Elastic Search Sink doc it is not specified which ElasticSearch and Opensearch versions are supported.
https://pulsar.apache.org/docs/en/io-elasticsearch-sink

### What is your suggestion?

Inform the users which versions are supported.

### Do you have any references?

ElasticSearch and Opensearch
https://pulsar.apache.org/docs/en/io-elasticsearch-sink

### Would you like to fix this issue?

no time

### Note

- [X] I have researched my question.

### PR DESCRIPTION
**Pull request by mendonk**
Fixes #15514

### CODE REVIEW

File: site2/docs/io-elasticsearch-sink.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ sidebar_label: Elasticsearch sink connector
...
 The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes.
 
+## Requirements
+
+To deploy an ElasticSearch sink connector, the following are required:
+
+- ElasticSearch 7 (ElasticSearch 8 is not currently supported in Pulsar 2.10)
+- OpenSearch 1.x
+
+The `compatibilityMode` configuration property specifies compatibility with ElasticSearch and OpenSearch clusters.
```
Comment (nicoloboschi) [inline site2/docs/io-elasticsearch-sink.md:16]: Unfortunately this feature is not released yet. Actually this is needed to handle ES 8.

File: site2/docs/io-elasticsearch-sink.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ sidebar_label: Elasticsearch sink connector
 
 The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes.
 
+## Requirements
+
+To deploy an ElasticSearch sink connector, the following are required:
+
+- ElasticSearch 7 (ElasticSearch 8 is not currently supported in Pulsar 2.10)
```
Comment (nicoloboschi) [inline site2/docs/io-elasticsearch-sink.md:13]: I would remove the note for ES 8

Comment (nicoloboschi) [reply site2/docs/io-elasticsearch-sink.md:13]: We could say that it will be supported in the future

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

**Review by nicoloboschi — APPROVED** [review]

**Review by Huanli-Meng — APPROVED** [review]
LGTM

File: site2/docs/io-elasticsearch-sink.md [hunk 3]
```diff
@@ -6,6 +6,12 @@ sidebar_label: Elasticsearch sink connector
 
 The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes.
 
+## Requirements
+
+To deploy an ElasticSearch sink connector, the following are required:
```
Comment (Anonymitaet) [inline site2/docs/io-elasticsearch-sink.md:11]:
```suggestion
To deploy an Elasticsearch sink connector, meet the following requirements.
```

**Review by Anonymitaet — COMMENTED** [review]
can you please rename your PR? eg. [improve][doc] xxx
guideline: https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#heading=h.wu6ygjne8e35

File: site2/docs/io-elasticsearch-sink.md (hunk 3 above)
Comment (Technoboy-) [reply site2/docs/io-elasticsearch-sink.md:11]: The original seems better.

Comment (Anonymitaet) [reply site2/docs/io-elasticsearch-sink.md:11]: 1. Elasticsearch, lower case for "s" is correct
2. The subject of "To deploy an Elasticsearch sink connector" is "you" (user), so the subject of the latter sentence should be "you" as well. The subject of "meet the following requirements." is you.

Comment (Technoboy-) [reply site2/docs/io-elasticsearch-sink.md:11]: > 1. Elasticsearch, lower case for "s" is correct

Sure

> 2. The subject of "To deploy an Elasticsearch sink connector" is "you" (user), so the subject of the latter sentence should be "you" as well. The subject of "meet the following requirements." is you.

I'm a little not agree.  I prefer the original.

**Pull request merged**